src/Dapr.AspNetCore/TwilioWebHookMiddleware.cs

```diff
@@ -0,0 +1,139 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Net.Http.Headers;
+    using System.Text;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.WebUtilities;
+    internal class TwilioWebHookMiddleware
+    {
+        private const string ContentType = "application/x-www-form-urlencoded";
+        private readonly RequestDelegate next;
+
+        public TwilioWebHookMiddleware(RequestDelegate next)
+        {
+            this.next = next;
+        }
+
+        public Task InvokeAsync(HttpContext httpContext)
+        {
+            // This middleware unwraps any requests with a cloud events (JSON) content type

```

Comment:
These comments are not applicable to Twillio, please add comments specific to this middleware.

Reply:
done